src/redux/books/books.js

```diff
@@ -0,0 +1,38 @@
+import { createSlice } from "@reduxjs/toolkit";
```

Comment:
- [ ] **Optional:** Great work learning and setting up React-Redox using the Redux toolkit. 👏👏

I strongly recommend taking out time to also learn the initial release AKA [React-Redux](https://react-redux.js.org/) still the official binding for react as this is what the current materials you are given are based on as well the requirement reviews and reviewers are guided with.